### PR TITLE
Task #11 Implement an endpoint of type POST

### PR DESCRIPTION
**Pull request by wjablnski**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,3 +1,4 @@
 NASA_API_KEY=
 NASA_API_URL=https://api.nasa.gov/neo/rest/v1/feed
+ROVER_API_URL=https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos
```
Comment (mdziarkach) [inline .env.example]: Do you think it might be worth separating the base URL and endpoints?

Comment (mdziarkach) [reply .env.example]: and it seems to me that it’s not worth storing endpoints in environment variables
I would leave only the base URL in env, and leave the rest in the config

Comment (wjablnski) [reply .env.example]: Probably yes

**Review by mdziarkach — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (wjablnski) [reply .env.example]: I could also add interceptor to avoid repeating base url in config, what do you think?

**Pull request merged**